Unable to build with php-src master

probably caused by https://github.com/php/php-src/pull/13108

CI build log:
```
...
#10 144.2  cc -I. -I/tmp/pear/temp/redis -I/tmp/pear/temp/pear-build-defaultuser11W7fq/redis-6.0.2/main -I/tmp/pear/temp/redis -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/usr/local/include/php/ext -I/tmp/pear/temp/pear-build-defaultuser11W7fq/redis-6.0.2/liblzf -I/tmp/pear/temp/redis/liblzf -DHAVE_CONFIG_H -g -O2 -D_GNU_SOURCE -DZEND_COMPILE_DL_EXT=1 -c /tmp/pear/temp/redis/redis_commands.c -MMD -MF redis_commands.dep -MT redis_commands.lo  -fPIC -DPIC -o .libs/redis_commands.o
#10 144.2  cc -I. -I/tmp/pear/temp/redis -I/tmp/pear/temp/pear-build-defaultuser11W7fq/redis-6.0.2/main -I/tmp/pear/temp/redis -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/usr/local/include/php/ext -I/tmp/pear/temp/pear-build-defaultuser11W7fq/redis-6.0.2/liblzf -I/tmp/pear/temp/redis/liblzf -DHAVE_CONFIG_H -g -O2 -D_GNU_SOURCE -DZEND_COMPILE_DL_EXT=1 -c /tmp/pear/temp/redis/library.c -MMD -MF library.dep -MT library.lo  -fPIC -DPIC -o .libs/library.o
#10 144.2  cc -I. -I/tmp/pear/temp/redis -I/tmp/pear/temp/pear-build-defaultuser11W7fq/redis-6.0.2/main -I/tmp/pear/temp/redis -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/usr/local/include/php/ext -I/tmp/pear/temp/pear-build-defaultuser11W7fq/redis-6.0.2/liblzf -I/tmp/pear/temp/redis/liblzf -DHAVE_CONFIG_H -g -O2 -D_GNU_SOURCE -DZEND_COMPILE_DL_EXT=1 -c /tmp/pear/temp/redis/redis.c -MMD -MF redis.dep -MT redis.lo  -fPIC -DPIC -o .libs/redis.o
#10 144.2  cc -I. -I/tmp/pear/temp/redis -I/tmp/pear/temp/pear-build-defaultuser11W7fq/redis-6.0.2/main -I/tmp/pear/temp/redis -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/usr/local/include/php/ext -I/tmp/pear/temp/pear-build-defaultuser11W7fq/redis-6.0.2/liblzf -I/tmp/pear/temp/redis/liblzf -DHAVE_CONFIG_H -g -O2 -D_GNU_SOURCE -DZEND_COMPILE_DL_EXT=1 -c /tmp/pear/temp/redis/redis_session.c -MMD -MF redis_session.dep -MT redis_session.lo  -fPIC -DPIC -o .libs/redis_session.o
#10 144.4 /tmp/pear/temp/redis/redis.c:30:10: fatal error: standard/php_random.h: No such file or directory
#10 144.4    30 | #include <standard/php_random.h>
#10 144.4       |          ^~~~~~~~~~~~~~~~~~~~~~~
#10 144.4 compilation terminated.
#10 144.4 /tmp/pear/temp/redis/library.c:59:10: fatal error: ext/standard/php_rand.h: No such file or directory
#10 144.4    59 | #include <ext/standard/php_rand.h>
#10 144.4       |          ^~~~~~~~~~~~~~~~~~~~~~~~~
#10 144.4 compilation terminated.
#10 144.4 make: *** [Makefile:209: redis.lo] Error 1
#10 144.4 make: *** Waiting for unfinished jobs....
#10 144.4 make: *** [Makefile:215: library.lo] Error 1
#10 146.8 ERROR: `make -j4' failed
```

--- a/backoff.c
+++ b/backoff.c
@@ -1,14 +1,14 @@
 #include "common.h"
 
+#if PHP_VERSION_ID < 80400
 #include <ext/standard/php_rand.h>
-
-#if PHP_VERSION_ID >= 70100
-#include <ext/standard/php_mt_rand.h>
 #else
+#include <ext/random/php_random.h>
+#endif
+
+#if PHP_VERSION_ID < 70100
 static zend_long php_mt_rand_range(zend_long min, zend_long max) {
-    zend_long number = php_rand();
-    RAND_RANGE(number, min, max, PHP_RAND_MAX);
-    return number;
+	return min + php_rand() % (max - min + 1)
 }
 #endif
 

--- a/library.c
+++ b/library.c
@@ -56,8 +56,13 @@
 #include <ext/json/php_json.h>
 #endif
 
-#include <ext/standard/php_rand.h>
 #include <ext/hash/php_hash.h>
+
+#if PHP_VERSION_ID < 80400
+#include <ext/standard/php_rand.h>
+#else
+#include <ext/random/php_random.h>
+#endif
 
 #define UNSERIALIZE_NONE 0
 #define UNSERIALIZE_KEYS 1

--- a/redis.c
+++ b/redis.c
@@ -27,12 +27,16 @@
 #include "redis_cluster.h"
 #include "redis_commands.h"
 #include "redis_sentinel.h"
-#include <standard/php_random.h>
 #include <ext/spl/spl_exceptions.h>
 #include <zend_exceptions.h>
 #include <ext/standard/info.h>
 #include <ext/hash/php_hash.h>
 
+#if PHP_VERSION_ID < 80400
+#include <ext/standard/php_random.h>
+#else
+#include <ext/random/php_random.h>
+#endif
 
 #ifdef PHP_SESSION
 #include <ext/session/php_session.h>

